### ISSUE TITLE
Put Darkmode Button in Header 

### ISSUE THREAD

**Issue by extemporary** [GitbookIO/public-docs#76]
The dark mode button for the docs is in the footer, where it might as well not exist unless you already know it's there, and is invisible unless you scroll to the very bottom

**Comment by scrivera-dugan** [comment]
Good idea — I've passed this along to the product team as a feature suggestion ✓ Closing this here since it's outside the scope of the docs repo, but thanks for sharing your feedback!
